test/language/expressions/tagged-template/invalid-escape-sequences.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2016 Tim Disney. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Invalid unicode escape sequence in tagged template literals are allowed
+es6id: 11.8.6
```

Comment:
esid is the preferred way going forward, so we don't need to keep updating section numbers. The named spec section is used instead, e.g. esid: sec-template-literals